README.md

```diff
@@ -12,6 +12,7 @@ All kustomize configs should be entirely declarative in nature. This means no ba
 
 - You need a running OCP 4.4 cluster and a valid KUBECONFIG.
 - You need at least one node with the `node-role.kubernetes.io/worker-cnf=""` label and a `MachineConfigPool` matching `worker-cnf` machine configurations
+- You need to install `jq` (a command line tool for parsing JSON) in the cluster.
```

Comment:
The command doesn't need to be inside in the cluster. it needs to be on local machine where you run `make deploy`